int64[] robot_num
float64 agent_radius
geometry_msgs/PoseArray obstacle_positions
geometry_msgs/Pose reference_position
float64[] obstacle_xwidth
float64[] obstacle_ywidth
